packages/contracts-bedrock/interfaces/L1/IL1StandardBridge.sol

```diff
@@ -3,9 +3,13 @@ pragma solidity ^0.8.0;
 
 import { IStandardBridge } from "interfaces/universal/IStandardBridge.sol";
 import { ICrossDomainMessenger } from "interfaces/universal/ICrossDomainMessenger.sol";
+import { ISystemConfig } from "interfaces/L1/ISystemConfig.sol";
 import { ISuperchainConfig } from "interfaces/L1/ISuperchainConfig.sol";
 
 interface IL1StandardBridge is IStandardBridge {
+
+    error ReinitializableBase_ZeroInitVersion();
```

Comment:
For customer errors please follow[ the new standard](https://github.com/ethereum-optimism/design-docs/blob/main/solidity/revert-strings-to-custom-error-migration.md) for their naming

Reply:
This is actually the correct error, it comes from `ReinitializableBase`